internal/logger/logger.go

```diff
@@ -133,6 +133,11 @@ func Info(v ...interface{}) {
 	defaultInfoLogger.Println(v...)
 }
 
+// Fatal calls the default info logger to print the message followed by a call to os.Exit(1).
```

Comment:
logger is not making a call to os.exit(1)

Reply:
also will it create any problem in doing os.exit(1) before the crash is happening?